great_expectations/util.py

```diff
@@ -1510,12 +1510,15 @@ def isclose(
         operand_a = operand_a.total_seconds()  # type: ignore[assignment]
         operand_b = operand_b.total_seconds()  # type: ignore[assignment]
 
-    return np.isclose(
-        a=np.float64(operand_a),  # type: ignore[arg-type]
-        b=np.float64(operand_b),  # type: ignore[arg-type]
-        rtol=rtol,
-        atol=atol,
-        equal_nan=equal_nan,
+    return cast(
```

Comment:
@Kilo59 What is the reason for the `cast` here?  According to documentation, `If both a and b are scalars, returns a single boolean value.` so we should be fine here, right?  Thanks.

Reply:
This method seems to return an alias of `bool` not the `bool` built-in. This confuses the type-checker.
```
great_expectations/util.py:1513: error: Incompatible return value type (got "bool_", expected "bool")  [return-value]
```

Reply:
@Kilo59 Oh yes, you are correct -- the `numpy` types sometimes have the same spelling as the `Python` types, but are in fact different objects (unfortunately, I must add).  Thanks for reminding of that.